source/manage-apps/deploy/compact-disk.txt

```diff
@@ -0,0 +1,66 @@
+.. _compact-disk:
+
+=======================
+Compact an Atlas Volume
+=======================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+
+Overview
+--------
+
+While Realm data compaction defragments the storage space the realm file uses 
+on a *client*, and also reduces the size of the __realm_sync database on Atlas 
+clusters, it does not reduce the overall size of the Atlas cluster. Compacting 
+the data on an Atlas volume involves several additional steps. 
+
+Steps to Compact and Reclaim Disk Space
+---------------------------------------
+
+You should use the following guidance for reducing Atlas disk space usage:  
+
+1. Enable Compaction on the Client
```

Comment:
```suggestion
1. Enable Compaction in App Services
```

This is more _for_ the client rather than _on_ the client given that you do it on the server side